openpype/hosts/maya/api/lib.py

```diff
@@ -2571,7 +2571,8 @@ def _get_attrs(node):
             new_name = "{0}_baked".format(short_name)
             new_node = cmds.duplicate(node,
                                       name=new_name,
-                                      renameChildren=True)[0]
+                                      renameChildren=True,
+                                      ic=True)[0]
```

Comment:
Please use the long flag name for clarity. As an example I first thought this might have been `ignoreChildren` instead of `inputConnections` without looking up maya documentation.
```suggestion
                                      inputConnections=True)[0]
```

Could you elaborate what attribute connections you were missing without this?

Reply:
Specifically Image planes were missing during default export (with `Bake Cameras to World-Space`)

Reply:
I think we are aiming more for regular cameras, not Perspective one..imho.